### ISSUE TITLE
PSReadLine cannot be started Preview extension MacOS

### ISSUE THREAD

**Issue by bgelens** [PowerShell/vscode-powershell#2623]
### System Details

System Details Output
```
### VSCode version: 1.44.0 2aae1f26c72891c399f860409176fe435a154b13 x64

### VSCode extensions:
2gua.rainbow-brackets@0.0.6
adamhartford.vscode-base64@0.1.0
amazonwebservices.aws-toolkit-vscode@1.8.0
andreaslenzen.vbscript-language-server-client@1.0.3
Azurite.azurite@3.6.0
ban.spellright@3.0.50
bbenoist.vagrant@0.5.0
DavidAnson.vscode-markdownlint@0.34.0
donjayamanne.githistory@0.6.3
eamodio.gitlens@10.2.1
EditorConfig.EditorConfig@0.14.5
Equinusocio.vsc-community-material-theme@1.4.1
Equinusocio.vsc-material-theme@32.5.0
equinusocio.vsc-material-theme-icons@1.1.4
eriklynd.json-tools@1.0.2
GrapeCity.gc-excelviewer@2.1.34
mauve.terraform@1.4.0
ms-azure-devops.azure-pipelines@1.165.1
ms-azuretools.vscode-azureappservice@0.16.4
ms-azuretools.vscode-azurefunctions@0.22.0
ms-azuretools.vscode-docker@1.0.0
ms-dotnettools.csharp@1.21.16
ms-kubernetes-tools.vscode-aks-tools@0.0.4
ms-kubernetes-tools.vscode-kubernetes-tools@1.1.1
ms-mssql.mssql@1.9.0
ms-python.python@2020.3.71659
ms-vscode.azure-account@0.8.9
ms-vscode.azurecli@0.5.0
ms-vscode.cpptools@0.27.0
ms-vscode.Go@0.13.1
ms-vscode.powershell@2020.3.0
ms-vscode.powershell-preview@2020.4.0
msazurermtools.azurerm-vscode-tools@0.8.5
PKief.material-icon-theme@4.0.1
redhat.vscode-xml@0.11.0
redhat.vscode-yaml@0.7.2
Shan.code-settings-sync@3.4.3
tht13.html-preview-vscode@0.2.5
twxs.cmake@0.0.17
Tyriar.shell-launcher@0.4.1
vitaliymaz.vscode-svg-previewer@0.5.1
vscoss.vscode-ansible@0.5.2
wayou.vscode-todo-highlight@1.0.4
wingrunr21.vscode-ruby@0.27.0
yzane.markdown-pdf@1.4.4


### PSES version: 2.1.0.0

### PowerShell version:

Name                           Value
----                           -----
PSVersion                      7.1.0-preview.1
PSEdition                      Core
GitCommitId                    7.1.0-preview.1
OS                             Darwin 19.4.0 Darwin Kernel Version 19.4.0: Wed Mar  4 22:28:40 PST 2020; root:xnu-6153.101.6~15/RELEASE_X86_64
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0


```

### Issue Description

Enabling verbose editorservice log level raises this warning:

`2020-04-09 08:37:49.496 +02:00 [WRN] PSReadLine unable to be loaded: Unable to find type [Microsoft.PowerShell.PSConsoleReadLine,Microsoft.PowerShell.PSReadLine2, Version=2.0.0.0, Culture=neutral, PublicKeyToken=null].`

I noticed as my keybindings stopped working (ctrl+r, etc) and coloring was not working anymore

#### Expected Behaviour

Should load

#### Actual Behaviour

Did not load

### Attached Logs

[1586414642-3e699a5d-aec5-4539-a3ab-3246af6d72241586413989494.zip](https://github.com/PowerShell/vscode-powershell/files/4454485/1586414642-3e699a5d-aec5-4539-a3ab-3246af6d72241586413989494.zip)

**Comment by PrzemyslawKlys** [comment]
I have same issue on Windows. After upgrade PSReadLine doesn't work anymore.

**Comment by TylerLeonhardt** [comment]
Looks like the assembly version changed...
https://github.com/PowerShell/PSReadLine/blob/master/PSReadLine/PSReadLine.csproj#L7

I'll get a fix out and try to push a preview today. In the meantime, you can downgrade the version of PSReadLine to 2.0.0 to be unblocked.

**Comment by bgelens** [comment]
@TylerLeonhardt PSReadline 2.1.0-beta1 is shipped with the preview extension. Downgrade would be difficult :)

**Comment by TylerLeonhardt** [comment]
If you install the 2.0.0 version of PSReadLine, it'll come _ahead_ of that path in the PSModulePath so it should work.

**Comment by bgelens** [comment]
I tried that already but it doesn't work like that apparently 

![image](https://user-images.githubusercontent.com/10071039/78904573-09ac2a00-7a7d-11ea-908f-59a4af674427.png)

`2020-04-09 16:12:27.160 +02:00 [WRN] PSReadLine unable to be loaded: Unable to find type [Microsoft.PowerShell.PSConsoleReadLine,Microsoft.PowerShell.PSReadLine2, Version=2.0.0.0, Culture=neutral, PublicKeyToken=null].`

**Comment by TylerLeonhardt** [comment]
Looks like you still have 2.0.1 in the first position. The assembly version in 2.0.1 was changed:

https://github.com/PowerShell/PSReadLine/blob/v2.0.1/PSReadLine/PSReadLine.csproj#L7

You need to grab 2.0.0 specifically to get it working, for now.

**Comment by bgelens** [comment]
As you can see in the screenshot, it is still loading 2.1.0-beta1 even though I don't have it in my own path anymore. 

Still I did as you asked and removed 2.0.1. The extension is still loading 2.1.0-beta1 though. See integrated terminal output:

![image](https://user-images.githubusercontent.com/10071039/78923637-d6779400-7a98-11ea-81a8-831914ce96a2.png)

The read square shows the currently loaded psreadline. So the workaround does not work. (Which is another issue on it's own?)

This is from the editorservices log:

```
2020-04-09 19:32:37.615 +02:00 [WRN] PSReadLine unable to be loaded: Unable to find type [Microsoft.PowerShell.PSConsoleReadLine,Microsoft.PowerShell.PSReadLine2, Version=2.0.0.0, Culture=neutral, PublicKeyToken=null].
```

**Comment by TylerLeonhardt** [comment]
Ah I guess we sort by version... Well, you can delete that folder and things will work again. I have a PR out to fix this.

**Comment by bgelens** [comment]
Thanks @TylerLeonhardt! Looking forward to the next preview release :)

**Comment by bgelens** [comment]
@TylerLeonhardt The workaround not working is indeed caused by sorting:  https://github.com/PowerShell/PowerShellEditorServices/blob/master/src/PowerShellEditorServices/Services/PowerShellContext/Session/PSReadLinePromptContext.cs#L36
